assets/js/src/core/modules/data-object/hooks/use-data-object-helper.ts

```diff
@@ -121,5 +128,37 @@ export const useDataObjectHelper = (): UseDataObjectReturn => {
     }
   }
 
-  return { openDataObject, executeDataObjectTask }
+  const formatPath = async (items: IFormatPathItems[], fieldName: string, dataObjectId: number): Promise<DataObjectFormatPathApiResponse> => {
+    const targets = items.reduce((acc, item) => {
+      acc[`object_${item.id}`] = {
+        id: item.id,
+        type: item.type,
+        label: item.fullPath,
+        path: item.fullPath,
+        nicePathKey: `object_${item.id}`
+      }
+      return acc
+    }, {})
+
+    const { data } = await store.dispatch(api.endpoints.dataObjectFormatPath.initiate({
+      body: {
+        objectId: dataObjectId,
+        targets,
+        context: {
```

Comment:
It will be a bit tricky but we need to somehow find a way to respect the different possible context cases:

https://github.com/pimcore/studio-backend-bundle/blob/75ac539df3485d772261d8645d50765710ac0647/src/DataObject/Legacy/PathFormatterHelper.php#L46